apps/etterlatte-saksbehandling-ui/server/src/routers/loggerRouter.ts

```diff
@@ -12,16 +12,44 @@ export interface IStackLineNoColumnNo {
   error: any
 }
 
+function stackInfoIsInvalid(numbers: IStackLineNoColumnNo): boolean {
+  return numbers.lineno < 1 || numbers.columnno < 0
+}
+
+const sourcemapLocation = '/app/client/assets'
+async function sourceMapMapper(numbers: IStackLineNoColumnNo): Promise<NullableMappedPosition> {
+  const sourcemapFile = fs.readdirSync(sourcemapLocation).find((file) => file.includes('.map')) ?? ''
+  const rawSourceMap = fs.readFileSync(`${sourcemapLocation}/${sourcemapFile}`).toString()
+  const smc = await new sourceMap.SourceMapConsumer(rawSourceMap)
+  return Promise.resolve(smc.originalPositionFor({ line: numbers.lineno, column: numbers.columnno }))
+}
+
+const GYLDIG_FNR = (input: string | undefined) => /^\d{11}$/.test(input ?? '')
+
+function findAndSanitizeUrl(url?: String): String {
+  if (url) {
+    const splittedUrl = url.split('/')
+    splittedUrl.map((urlpart) => {
+      if (GYLDIG_FNR(urlpart)) {
+        return urlpart.substring(0, 5)
+      }
+      return urlpart
+    })
+  }
+  return ''
+}
```

Comment:
Ville heller fikset rotproblemet her, som er at vi fortsatt bruker url med fnr. Vi kan be logg-gjengen om å fjerne de tilfelle hvor skaden allerede har skjedd.

Reply:
Nvm! Hadde helt glemt at det må være sånn i frontend. 
Men ville da heller sensurert fnr i stedet, slik at url-oppbygging fortsatt er korrekt. 

Eks: `https://etterlatte-saksbehandling.intern.dev.nav.no/person/211212*****`